eng/TestMSBuild.targets

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <!--
+  Copies the NuGet dll's required for testing to testing folders
+   -->
+  <Target Name="CopyNuGetDependencies" AfterTargets="ResolvePackageAssets">
+    <ItemGroup>
+        <_NuGetDependencies Include="$(RepoRoot).dotnet\sdk\$(DotNetCliVersion)\Sdks\Microsoft.NET.Sdk\tools\net472\NuGet.Frameworks.dll" /> 
```

Comment:
Should these instead have a dev-only (`PrivateAssets=all`?) reference to NuGet packages, and let that copy?